api/migrations/0001_initial.py

```
@@ -16,6 +16,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('auth', '0012_alter_user_first_name_max_length'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
     ]
```

Ruff (0.14.13)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)